src/Sylius/Bundle/ApiBundle/Command/Account/ResendVerificationEmail.php

```diff
@@ -16,30 +16,24 @@
 use Sylius\Bundle\ApiBundle\Command\ChannelCodeAwareInterface;
 use Sylius\Bundle\ApiBundle\Command\IriToIdentifierConversionAwareInterface;
 use Sylius\Bundle\ApiBundle\Command\LocaleCodeAwareInterface;
+use Sylius\Bundle\ApiBundle\Command\ShopUserIdAwareInterface;
 
 /** @experimental */
-class ResendVerificationEmail implements ChannelCodeAwareInterface, LocaleCodeAwareInterface, IriToIdentifierConversionAwareInterface
+class ResendVerificationEmail implements ShopUserIdAwareInterface, ChannelCodeAwareInterface, LocaleCodeAwareInterface, IriToIdentifierConversionAwareInterface
 {
-    /** @var string */
-    public $email;
+    /** @var mixed */
+    public $shopUserId;
 
     /**
-     * @var string
-     * @psalm-immutable
+     * @var string|null
      */
     public $channelCode;
```

Comment:
It's PHP 7.4 baby 😂 
```suggestion
    public ?string $channelCode = null;
```

Reply:
Tempting, but I'm not sure, we should do it before the main effort. /cc @Zales0123 @Arminek @GSadee